src/sentry/integrations/jira/notify_action.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import absolute_import
+
+import logging
+import sentry_sdk
+
+from django import forms
+
+from sentry.models import Integration
+from sentry.rules.actions.base import EventAction
+from sentry.utils import metrics
+
+logger = logging.getLogger("sentry.rules")
+
+
+class JiraNotifyServiceForm(forms.Form):
+    # TODO 1.0 Add form fields.
+
+    def __init__(self, *args, **kwargs):
+        super(JiraNotifyServiceForm, self).__init__(*args, **kwargs)
+
+    def clean(self):
+        return super(JiraNotifyServiceForm, self).clean()
+
+
+class JiraCreateTicketAction(EventAction):
+    form_cls = JiraNotifyServiceForm
+    label = u"TODO Create a {name} Jira ticket"
+    prompt = "Create a Jira ticket"
+
+    def __init__(self, *args, **kwargs):
+        super(JiraCreateTicketAction, self).__init__(*args, **kwargs)
+        # TODO 1.1 Add form_fields
+        self.form_fields = {}
+
+    def is_enabled(self):
+        # TODO move to superclass
```

Comment:
@mgaeta do you mean the `EventAction` class?

Reply:
I made a intermediate class `IntegrationEventAction` that DRYs up some of the logic. If it's overkill, I'll revert.